packages/rancher-logging/package.yaml

```diff
@@ -1,5 +1,5 @@
 url: oci://ghcr.io/kube-logging/helm-charts/logging-operator:4.4.0
-version: 104.0.0-rc1
+version: 104.0.0-rc2
```

Comment:
We've been working with the same rc number before the release (a little counter intuitive I know). Can you move these changes to 104.0.0-rc1 instead please.

Otherwise looks good to me.

Reply:
Ah okay, will do

Reply:
Should be good now. I've done the same change for the 2.8 PR